native/cuda/main.cc

```diff
@@ -1,10 +1,9 @@
 #include <opencv2/cudaobjdetect.hpp>
 
-#include "opencv-gpu.h"
-#include "opencv-wrapper.h"
-#include "utils.h"
+#include "../utils.h"
+#include "main.h"
```

Comment:
main suggests that there might be a main function and this is a binary executable.

we could do `native/cuda/cuda.[h|cc]`